pkgs/by-name/wr/wrangler/package.nix

```diff
@@ -30,15 +30,16 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     llvmPackages.libcxx
     llvmPackages.libunwind
-    musl
```

Comment:
Still not sure why it was here. A few packages mention it - for example:
```
lib/node_modules/.pnpm/@types+node@20.8.3/node_modules/@types/node/ts4.8/fs.d.ts:         * On Linux, when Node.js is linked against musl libc, the procfs file system must
```
But it doesn't look like the nixos version is using musl at all.